recipe/meta.yaml

```diff
@@ -22,8 +22,8 @@ outputs:
       run:
         - python >=3.8
         - cartopy >=0.20
-        - cf-units >=3
-        - cftime >=1.5.0 
+        - cf-units =3.0
+        - cftime <1.6.0
```

Comment:
This would pick up cftime 1.5.2 which I believe still has a failing test:
```
======================================================================
FAIL: test_num2date (cf_units.tests.test_unit.TestNumsAndDates)
----------------------------------------------------------------------
Traceback (most recent call last):
  .../site-packages/cf_units/tests/test_unit.py", line 956, in test_num2date
    self.assertEqual(res.calendar, "gregorian")
AssertionError: 'standard' != 'gregorian'
- standard
+ gregorian
```

Don't you need to pin to 1.5.1.1?

Reply:
👀

https://github.com/Unidata/cftime/blob/924c90377a857843661ae36eba1cd1a6c7f338c3/Changelog#L14-L17

Reply:
Yeah, that's a bugfix...